### PR TITLE
Cleanup Regex Rules

### PR DESCRIPTION
**Pull request by Sarabveer**
- [x] Cleanup Regex Rules
- [x] Add missing ASCII mappings

### CODE REVIEW

File: lib/unicodeMappings.json [hunk 1]
```diff
@@ -32,6 +33,7 @@
   "®": "੍ਰ",
   "´": "ੵ",
   "µ": "ੰ",
+  "μ": "ੰ",
```
Comment (Harjot1Singh) [inline lib/unicodeMappings.json]: Is this not duplication?

File: lib/unicodeMappings.json [hunk 2]
```diff
@@ -103,5 +103,6 @@
   "\\": "ਞ",
   "Ç": "☬",
   "¡": "ੴ",
-  "æ": "਼"
+  "æ": "਼",
+  "‚": "❁"
 }
```
Comment (Harjot1Singh) [inline lib/unicodeMappings.json]: Add newline to end of file

File: lib/unicodeMappings.json (hunk 1 above)
Comment (Sarabveer) [reply lib/unicodeMappings.json]: If you inspect both chars, they are different (U+00B5 and U+03BC respectively) but are still assigned in the ascii font.

Comment (Harjot1Singh) [reply lib/unicodeMappings.json]: Ah, thought something like this might be the case 👍

File: lib/unicodeMappings.json (hunk 2 above)
Comment (Sarabveer) [reply lib/unicodeMappings.json]: Done

**Comment by Harjot1Singh** [comment]
LGTM. Place #Patch somewhere in the merge title message

**Pull request merged**